data/human/rim jobs.txt

```diff
@@ -100,9 +100,11 @@ mission "Han Sizer Month [0]"
 		distance 5 10
 		attributes "rim"
 		not attributes "station"
+	on enter
+		dialog `As you enter the system your passengers gather together to stare out the viewport. One of them pulls out a datapad and reads what sounds like a log entry by Han Sizer, describing the initial exploration of the system.`
```

Comment:
Seems like it would be annoying for this `on enter` to pop up every time. I don't think this is the best way to give the player this info.

Reply:
That's why I only did it for *one* of these missions. So not every one of them is going to get this pop up, only some.

Reply:
As I said in the review description, I'd rather just have a Remembrance Day-like mission conversation. Even if it's only one of the missions that has this on enter, it's still _every time_ you get this mission that it's going to appear.

Reply:
The `on enter` has been replaced with an on stopover, so its timing makes much better sense.

Reply:
Still not what I asked for, though.

Reply:
But it is now on par with, say, the description of attending a research conference job, or the various retrieval jobs. It's not an onerous amount of text, and it *is* appropriate for what the job is. If it's a huge annoyance, than players can simply not do it - it's not something they are being forced to do. It's not even super profitable. It's an atmospheric/setting job that conveys a bit of culture of the region, and as such should be conveying a bit of what they're doing.

Having an additional mission as per your suggestion would be a great idea, and I'll be thinking about doing that. That is, however, a separate thing that I will aim to have in a standalone PR, and not a part of making these jobs a little more meaningful of what they are intended to convey.

Reply:
These are more complex jobs than the remembrance day things, and as such deserve the minute increase in screen time presented by a single `on stopover.` Also unlike the remembrance day jobs, the entire point of the job is the travel. They're not hiring the player to get from A to B, they're hiring the player to go through C. As such, it is appropriate that something actually happens while they are in transit.

As far as that goes, I don't really think the remembrance day jobs are a good model, because those are effectively meaningless. They have less character or impact than the "take a care package of X tons of Y stuff to a student on Z" jobs, and much less memorable. The Remembrance Day jobs are a step up from generic jobs, sure, but not by much. One explanatory conversation does *not* have any significant impact on justifying, explaining, or making meaningful all the varius remembrance day jobs I've done. They're all just bland "whatever" things that are basically indistinguishable from all the rest of the timed jobs. But that's a problem for another day, in a different PR.

Right now, I'm trying to take a small step in making *these* jobs slightly more relevant by including a tiny bit of flavor text, directions to where the player can find more, and a single `on stopover` in *one* of the jobs that illustrates that they're actually doing something on this trip. It's not just a "X passengers are taking up bunks while you wander around and happen to pass through a bunch of systems before they get off again."